source/tutorials/profiling-gpu/profiling-gpu.rst

```diff
@@ -0,0 +1,134 @@
+Profiling GPU
+########################################################################
+
+HW pre-requisites
+===========================
+Build machine/PC with Ubuntu 16.04
```

Comment:
I think the rendering will look more organized if we put these prerequisites in a bulleted list.